lnrpc/rpc.proto

```diff
@@ -2093,11 +2093,13 @@ message PendingChannelsResponse {
         */
         int64 remote_chan_reserve_sat = 7;
 
-        // True if we initiated opening the channel.
-        bool initiator = 8;
+        reserved 8;
```

Comment:
If this hasn't been released yet, I think we just stick with record 8 for the enum.